### PR TITLE
DFA-2290 - Create the 'Email invite sent' page

### PR DESCRIPTION
**Pull request by BorisP-GDS**
Create the 'Email invite sent' page front-end

### CODE REVIEW

File: express/src/views/service-details/invitation-sent.njk [hunk 1]
```diff
@@ -0,0 +1,30 @@
+{% extends "layout/base.njk" %}
```
Comment (CharlesIC) [inline express/src/views/service-details/invitation-sent.njk:1]:
```suggestion
{% extends "./layout-service-details.njk" %}
```

This should be using the service details layout

File: express/src/views/service-details/invitation-sent.njk [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+          <li class="govuk-breadcrumbs__list-item govuk-!-font-size-19 govuk-!-font-weight-bold">{{ serviceName }}</li>
+        {% endif %}
+      </ol>
+      <a href="/change-service-name/{{ serviceId }}/{{ selfServiceClientId }}/{{ clientId }}?serviceName={{ serviceName }}"
+         class="govuk-link govuk-link--no-visited-state navigation-service-change govuk-!-font-size-19">
+        Change service name
+      </a>
+    </div>
+  </div>
+{% endblock %}
```
Comment (CharlesIC) [inline express/src/views/service-details/invitation-sent.njk:23]: Introduce a variable to the `layout-service-details.njk` template to hide the sidebar if needed and avoid duplicating the "Your services > ..." heading

**Review by CharlesIC — COMMENTED** [review]
Some changes needed to avoid duplication

File: express/src/views/service-details/invitation-sent.njk (hunk 2 above)
Comment (BorisP-GDS) [reply express/src/views/service-details/invitation-sent.njk]: Updated.

**Review by JohnWattsGDS — APPROVED** [review]

**Pull request merged**